pages/index.js

```diff
@@ -60,4 +30,46 @@ const Button = styled.button`
   border-radius: 3px;
   margin: 2px;
   padding: 0.5px;
+  display: center;
+  text-align: center;
+`;
+
+const StyledLink = styled.p`
+  display: center;
+  text-align: center;
+  font-size: 2em;
+  color: blue;
 `;
+
+export default function Landingpage() {
+  const router = useRouter();
+  const [output, setOutput] = useState("");
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const entry = event.target.elements.text.value;
+    setOutput(entry);
+  };
+  return (
+    <>
+      <Title>Your Travel Journal</Title>
+      <Form onSubmit={handleSubmit}>
+        <fieldset>
+          <legend>Where is your journey going?</legend>
+          <Label htmlFor="text">City or Country?</Label>
+          <input
+            type="text"
+            required
+            id="text"
+            name="text"
+            placeholder="enter your destination..."
+          />
+          <Button type="submit">Submit</Button>
+          <p>{output}</p>
+        </fieldset>
+      </Form>
+      <Link href="/EntryListPage">
+        <StyledLink>Your Entries</StyledLink>
+      </Link>
```

Comment:
Better extend that Link component with styled-components

Reply:
i just removed it, as i want to change it later anyway - but  i keep this way to handle with double components in mind for the styling, thx